src/utilities/flexbox.scss

```diff
@@ -43,12 +44,6 @@
     .flex#{$variant}-self-baseline    { align-self: baseline !important; }
     .flex#{$variant}-self-stretch     { align-self: stretch !important; }
 
-    // Shorthand for equal width and height cols
-    .flex#{$variant}-item-equal {
-      flex-grow: 1;
-      flex-basis: 0;
```

Comment:
Replacing

```css
.flex-item-equal {
  flex-grow: 1;	
  flex-basis: 0;
}
```

with

```css
.flex-1 { flex: 1; }
```

should be ok, since `flex-shrink: 1` is the default for flex items and `flex: 1` gets computed as:

 ```css
.flex-1 {
  flex-grow: 1;
  flex-shrink: 1;
  flex-basis: 0%;
}
```